linux/src/threadpool_linux.c

```diff
@@ -24,7 +24,8 @@
 
 #include "c_pal/threadpool.h"
 
-#define DEFAULT_TASK_ARRAY_SIZE 2048
+#define DEFAULT_TASK_ARRAY_SIZE         2048
+#define THREADPOOL_SEMEPHORE_TIMEOUT    1
```

Comment:
>THREADPOOL_SEMEPHORE_TIMEOUT [](http://example.com/codeflow?start=8&length=28)

THREADPOOL_SEMAPHORE_TIMEOUT_S or ...SECONDS?

Reply:
Is 1 second small enough? We could use `tv_nsec` instead and make it like 100ms or something

Reply:
I can try that out, I wasn't sure how small an interval it should be.